examples/Batch/WordCountFSharp/App.config

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!--*************************************************************************************-->
+  <!-- Log4Net is used by the example -->
+  <!--**************************************************************************************-->
+  <configSections>
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
+  </configSections>
+  <log4net>
+    <root>
+      <level value="DEBUG" />
+      <appender-ref ref="ConsoleAppender" />
+      <!--<appender-ref ref="LogFileAppender" />-->
+    </root>
+    <appender name="ConsoleAppender" type="log4net.Appender.ConsoleAppender">
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="[%date] [%thread] [%-5level] [%logger] - %message%newline" />
+      </layout>
+    </appender>
+    <appender name="LogFileAppender" type="log4net.Appender.RollingFileAppender">
+      <file type="log4net.Util.PatternString">
+        <conversionPattern value="%env{TEMP}\\SparkCLRLogs\\SparkCLR-WordCount_%env{COMPUTERNAME}[%processid].log" />
+      </file>
+      <param name="AppendToFile" value="true" />
+      <param name="MaxSizeRollBackups" value="2000" />
+      <param name="MaxFileSize" value="51200000" />
+      <param name="StaticLogFileName" value="false" />
+      <param name="DatePattern" value=".yyyy_MM_dd_hh" />
+      <param name="RollingStyle" value="Composite" />
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="[%date] [%thread] [%-5level] [%logger] - %message%newline" />
+      </layout>
+    </appender>
+  </log4net>
+  <appSettings>
+    <!--********************************************************************************************************-->
+    <!--** Uncomment the following settings to run Spark driver executable in **local** or **debug** modes ** -->
+    <!--** In debug mode, the driver is not launched by CSharpRunner but launched from VS or command prompt not configured for SparkCLR ** -->
+    <!--** CSharpBackend should be launched in debug mode as well and the port number from that should be used below ** -->
+    <!--** Command to launch CSharpBackend in debug mode is "sparkclr-submit.cmd debug" ** -->
+    <!--********************************************************************************************************-->
+    <add key="CSharpWorkerPath" value="C:\Users\Isaac\Source\Repos\Mobius\examples\Batch\WordCount\bin\Debug\CSharpWorker.exe"/>
```

Comment:
Keeping these two settings uncommented will break the app in the non-debug mode. Please comment them out.